src/gasman.c

```diff
@@ -324,7 +324,7 @@ void CHANGED_BAG_IMPL(Bag bag) {
 #define SpaceBetweenPointers(a,b) (((UInt)((UInt)(a) - (UInt)(b)))/sizeof(Bag))
 
 #define SizeMptrsArea SpaceBetweenPointers(OldBags, MptrBags)
-#define SizeOldBagsArea SpaceBetweenPointers(YoungBags,OldBags)
+#define SizeOldBagsArea SpaceBetweenPointers(YoungBags, OldBags)
 #define SizeYoungBagsArea SpaceBetweenPointers(AllocBags, YoungBags)
 #define SizeAllocationArea SpaceBetweenPointers(StopBags, AllocBags)
 #define SizeUnavailableArea SpaceBetweenPointers(EndBags, StopBags)
```

Comment:
BTW, I don't quite understand the name `SizeUnavailableArea`.  Isn't this rather *available* space? Resp. free, unused, ... ? @stevelinton @ChrisJefferson what am I missing?